website/docs/r/app_service.html.markdown

```diff
@@ -189,7 +189,7 @@ A `site_config` block supports the following:
 
 * `ip_restriction` - (Optional) A [List of objects](/docs/configuration/attr-as-blocks.html) representing ip restrictions as defined below.
 
-* `java_version` - (Optional) The version of Java to use. If specified `java_container` and `java_container_version` must also be specified. Possible values are `1.7`, `1.8` and `11`.
+* `java_version` - (Optional) The version of Java to use. If specified `java_container` and `java_container_version` must also be specified. Possible values are `1.7`, `1.8` and `11` and their minor/patch versions.
```

Comment:
could we give an example of with a patch version, since it's slightly different to regular semver:

```suggestion
* `java_version` - (Optional) The version of Java to use. If specified `java_container` and `java_container_version` must also be specified. Possible values are `1.7`, `1.8` and `11` and their minor/patch versions (e.g. `1.7.0_80`).
```